README.md

```diff
@@ -114,12 +117,76 @@ in realtime: they both erase the file with their data.
 
 So make sure you're modifying it from one location at a time with the latest up-to-date Todo.txt file.
 
+## Supported auth backends
+
+| Name | Configuration value | Additional PyPI dependencies |
+|------|---------------------|------------------------------|
+| Predefined users | `DictAuth` |  |
+| WebDAV auth | `WebDavAuth` | `webdavclient3` |
+
+The `DictAuth` uses the `USERS` dict from the config as user database. With this nothing fancy happens
+at all. The `WebDavAuth` has no local user database. It forwards the login data from the user to the
+configured webdav server and tries to log in on his behalf. If this is successful, access will be granted.
+
 ## Supported storage backends
 
 | Name | Configuration value | Additional PyPI dependencies |
 |------|---------------------|------------------------------|
 | Local file system | `FileSystem` |  |
 | [Dropbox](https://www.dropbox.com/) | `Dropbox` | `dropbox` |
+| WebDAV | `WebDav` | `webdavclient3` |
+
+## Multi user support
+
+The `WebDav` storage backend has two special features:
+* The path of the todo.txt file can contain a placeholder for the user in the form `{username}` which will
+  be replaced by the current user name. With this its possible to have multiple todo files for multiple
+  users.
+* The actual credentials of the user will be either taken from the backends config values `webdav_login`
+  and `webdav_password` or, if omitted, from the current user.
+
+With these features there are several multi-user-scenarios possible:
+* You can have predefined users in the `USERS` dict that share one and the same todo file (no user 
+  placeholder, dav credentials given in the storage config and `DictAuth`).
+* You can have predefined users in the `USERS` dict which each has its own todo.txt file on the same
+  storage (filename with user placeholder, dav credentials given in the storage config and `DictAuth`)
+* You can have all the above using your webdav server as an auth backend by using the `WebDavAuth`.
+* You can have each webdav user have its own file on its own storage by using the placeholder and
+  omitting the dav credentials from the storage config.
+
+## Nextcloud Usage
```

Comment:
Are you sure Nextcloud usage needs its very own readme section? I mean, WebDAV isn't a standardized protocol which may be used by plenty of systems?

Reply:
Its mainly to give people an easy starting point what to put into the path setting. More documentation is always better I think :).

Reply:
Okay then, NVM :)